cluster/src/main/java/io/atomix/cluster/ClusterMembershipService.java

```diff
@@ -60,4 +61,17 @@ default Member getMember(String memberId) {
    */
   Member getMember(MemberId memberId);
 
+  /**
+   * Returns a member by address.
+   *
+   * @param address the member address
+   * @return the member or {@code null} if no member with the given address could be found
+   */
+  default Member getMember(Address address) {
```

Comment:
isn't better to just return `Optional<Member>`?

Reply:
That's probably a good idea. `Optional` is underused in the interfaces. Should do that in a separate PR though